reflekt-plugin/src/test/kotlin/io/reflekt/plugin/analysis/ReflektUsesTestModel.kt

```diff
@@ -0,0 +1,58 @@
+package io.reflekt.plugin.analysis
+
+import org.jetbrains.kotlin.psi.KtNamedFunction
+
+/*
+* There are helper data classes to parse JSON.
+* TODO: I am not sure that it is the best way, but I did not find a better way.
+* */
+
+data class SubtypesToUsesTest(
+    val subtypes: Set<String> = emptySet(),
+    val uses: MutableList<String> = mutableListOf()
```

Comment:
Can't we just use MutableSet<String> here so we don't need to write functions to compare lists as sets?